common/components/ui/Identicon.tsx

```diff
@@ -12,31 +13,12 @@ export default function Identicon(props: Props) {
   // FIXME breaks on failed checksums
   const identiconDataUrl = isValidETHAddress(props.address) ? toDataUrl(props.address) : '';
   return (
-    <div style={{ position: 'relative', width: size, height: size }} title="Address Identicon">
-      <div
-        style={{
-          position: 'absolute',
-          left: 0,
-          right: 0,
-          top: 0,
-          bottom: 0,
-          borderRadius: '50%',
-
-          boxShadow: `
-                    inset rgba(255, 255, 255, 0.5) 0 2px 2px,
-                    inset rgba(0, 0, 0, 0.6) 0 -1px 8px
-                  `
-        }}
-      />
```

Comment:
This by all accounts should be a good change, but (not your fault for missing this, it's a hack) inline styling was because this component was being used for the printable paper wallet, since the window that pops up for it doesn't use CSS. Honestly I want to convert that thing to use _no components_ and be entirely self-contained because of how goddamn annoying it is as a use-case. I'll probably go through and make a PR for that at another point.

But suffice it to say, these need to stay linine for a bit.